shared/django_apps/rollouts/models.py

```diff
@@ -28,6 +42,24 @@ class FeatureFlag(models.Model):
         help_text="Values are between 0 and 1. Eg: 0.5 means 50% of users",
     )
     salt = models.CharField(max_length=32, default=default_random_salt)
+    platform = models.CharField(
+        max_length=1, choices=Platform.choices, default=Platform.BACKEND
+    )
+    # Represents if an experiment has been cleaned up and
+    # is no longer running anymore
+    is_active = models.BooleanField(
+        default=True,
+        help_text="This should be on if the experiment is currently running. Otherwise turn it off if the experiment has finished and is cleaned up",
+    )
+
+    # The field we're rolling out over. Users with the same identifier
+    # will always receive the same variant. EG: if you rollout over org_id,
+    # then users in the same org see the same variant
+    rollout_identifier = models.CharField(
+        max_length=1,
+        choices=RolloutIdentifier.choices,
+        default=RolloutIdentifier.OWNER_ID,
+    )
```

Comment:
i think `rollout_universe` feels a little clearer to me than `rollout_identifier` but not a strong preference